src/widgets/Cell.php

```diff
@@ -302,21 +321,16 @@ private function generateCellControls()
             ['label' => $this->id]
         ];
 
-        $templates = WidgetTemplate::find()
-            ->where(['php_class' => TwigTemplate::class])
-            ->andWhere(['hide_in_list_selection' => WidgetTemplate::IS_VISIBLE_IN_LIST])
-            ->orderBy('name')
-            ->all();
-        foreach ($templates as $template) {
+        foreach (self::widgetTemplateItems() as $templateId => $templateName) {
```

Comment:
The method `widgetTemplateItems()` is not static and should therefore be called here via `$this->widgetTemplateItems()`, or am I missing something?